### DISCUSSION TITLE
Can we have the app in Fdroid?

### DISCUSSION THREAD

**Discussion by ticoombs** [advplyr/audiobookshelf-app#171] [General]
Hi advplyr,

Would it be okay if I put in a request to have Audiobookshelf including in the Fdroid repository? 

I currently do not have Play Services on some of my devices and wish to have this app on those devices if possible. I'd like to put it in the FDroid repository if that is okay.

(I'm happy to package and maintain the fdroid, I've already fell in love with audiobookshelf)

Cheers

**Comment by advplyr** [comment]
I have looked into it briefly and thought it would be better to start uploading to fdroid once the app makes it out of beta. It looked like a lot of additional work for releases to keep fdroid up-to-date. If you want to take that on it would be helpful.
You could join the discord to discuss it

**Comment by 10leej** [comment]
You can setup your own F-Droid repository if you want a faster release model.
